__tests__/suites/paypal-button-click-pom.test.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from "chai";
+import { CheckoutPage } from "../pages/checkoutPage";
+import { HomePage } from "../pages/homePage";
+
+describe("Testing Paypal buttons", () => {
+
+    beforeEach('Open StackDemo', () => {
+        browser.url("https://developer.paypal.com/demo/checkout/#/pattern/client");
+    });
+
+    afterEach('clear sessionstorage', () => {
+        browser.execute(() => sessionStorage.clear())
+    });
+
+    it("should start payment flow when clicking on paypal button", () => {
+        const homePage = new HomePage();
+        homePage.switchToPaypalFrame();
+        homePage.clickPaypalButton();
+        homePage.switchToParentFrame();
```

Comment:
can these be business function oriented?
just one function which performs the overall activity and moves ahead